index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" href="http://fonts.googleapis.com/css?family=Roboto">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <header class="navbar">
+    <a class="logo" href="#">Marco</a>
+    <img class="hamburger" src="images/hamburger.png" alt="">
```

Comment:
- Hey, in order to enforce the use of a semantic tag, kindly wrap the image in a nav tag as the nav should contain the introductory links in your webpage.